client/components/ovh-task-follow/ovh-task-follow.component.controller.js

```diff
@@ -0,0 +1,100 @@
+angular.module("managerApp").controller("ovhTaskFollowCtrl", function ($scope, $q, $http, $interval, $uibModal, $translate, $translatePartialLoader) {
+    "use strict";
+
+    var self = this;
+    var taksInfoPollPromise = null;
+
+    self.data = {
+        alert: null,
+        tasks: null
+    };
+
+    self.loading = {
+        init: false
+    };
+
+    /*= ==============================
+    =            HELPERS            =
+    ===============================*/
+
+    function getTaskInfo () {
+        return $http.get("/ovh-tasks", {
+            serviceType: "aapi"
+        }).then(function (response) {
+            if (_.isEmpty(response)) {
+                self.data.alert = null;
+                self.data.tasks = [];
+            } else {
+                self.data.alert = _.get(response, "data.alert", "");
+                self.data.tasks = _.map(_.get(response, "data.tasks", []), function (task) {
+                    task.comments = _.map(task.comments, function (comment) {
+                        comment.comment_text = _.get(comment, "comment_text", "").replace(/\\'/g, "'").replace(/\\"/g, "\"");
```

Comment:
it's hard to tell what you're trying to accomplish here at first glance.  Use a function maybe?